### PR TITLE
Add 2023 Oregon tax rules

### LINKED ISSUE #3871: Add 2023 Oregon tax rules
Reference: [2023 Oregon Income Tax Form OR-40 Instructions](https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2023.pdf)
[2023 Oregon OR-17 Individual Income Tax Guide](https://www.oregon.gov/dor/forms/FormsPubs/publication-or-17_101-431_2023.pdf)

### PR DESCRIPTION
**Pull request by jialyu0411**
Fixes #3871

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- bump: minor
+  changes:
+    added:
+    - Oregon ctc credit.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Oregon 2023 parameter values. 
```

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- bump: minor
+  changes:
+    added:
+    - Oregon ctc credit. 
+    - Oregon exemption credit. 
+    - Oregon kicker credit.
+    - Oregon retirement income credit.
+    - Oregon standard deduction.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:8]:
```suggestion
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml [hunk 3]
```diff
@@ -2,6 +2,7 @@ description: Oregon returns this portion of the filer's prior-year tax liability
 values:
   2021-01-01: 0.17341
   2022-01-01: 0
+  2023-01-01: 0.4428
 metadata:
   unit: /1
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:7]:
```suggestion
  unit: /1
  period: year
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:7]: could you check other files for missing period

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml [hunk 4]
```diff
@@ -1,9 +1,12 @@
 description: Oregon reduces its Child Tax Credit for filers with Oregon adjusted gross income exceeding this amount.
 values:
   0000-01-01: 25_000
+  2023-01-01: 30_000
 metadata:
   unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml:6]:
```suggestion
  unit: currency-USD
  period: year
```

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: could you add the 2023 reference to all the yaml files which we have in OR to show that the parameter does not change as well

File: policyengine_us/parameters/gov/states/or/tax/income/credits/exemption/amount.yaml [hunk 5]
```diff
@@ -3,6 +3,7 @@ values:
   # Defined in the law as $90 in 1986, indexed by CPI-U.
   2021-01-01: 213
   2022-01-01: 219
+  2023-01-01: 236
 metadata:
   unit: currency-USD
   label: OR exemption amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/exemption/amount.yaml:9]:
```suggestion
  label: Oregon exemption amount
  period: year
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml [hunk 6]
```diff
@@ -2,6 +2,7 @@ description: Oregon returns this portion of the filer's prior-year tax liability
...
   label: OR kicker rate
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:8]:
```suggestion
  label: Oregon kicker rate
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/or/tax/income/credits/kicker/percent.yaml:8]: change that in all parameters

File: policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml [hunk 7]
```diff
@@ -2,6 +2,7 @@ description: Oregon provides a minimum standard deduction of this amount for fil
 values:
   2021-01-01: 1_100
   2022-01-01: 1_150
+  2023-01-01: 1_250
 metadata:
   unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml:7]:
```suggestion
  unit: currency-USD
  period: year
```

File: policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml [hunk 8]
```diff
@@ -2,6 +2,7 @@ description: Oregon provides a minimum standard deduction of this amount for fil
...
   label: OR minimum deduction if claimable as dependent
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml:8]:
```suggestion
  label: Oregon minimum deduction if claimable as dependent
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml [hunk 9]
```diff
@@ -4,6 +4,7 @@ values:
   2023-01-01: 1_000
 metadata:
   unit: currency-USD
+  period: year
   label: Oregon Child Tax Credit amount
   reference:
     - title: 2023R1 House Bill 3235 Section 2 (3)(a)
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml:10]: is this on the 2023 tax form?

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml:10]: same question for all ctc files

File: policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/match/has_young_child.yaml [hunk 10]
```diff
@@ -3,11 +3,18 @@ values:
...
     - title: 2021 Oregon Income Tax Form OR-40 Instructions
       href: https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2021.pdf#page=18
     - title: 2022 Oregon Income Tax Form OR-40 Instructions
       href: https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2022.pdf#page=18
+    - title: 2023 Oregon Income Tax Form OR-40 Instructions
+      href: https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2023.pdf#page=18
+
+
+      
+
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/match/has_young_child.yaml:20]: delete extra lines

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml [hunk 11]
```diff
@@ -1,9 +1,13 @@
 description: Oregon reduces its Child Tax Credit for filers with Oregon adjusted gross income exceeding this amount.
 values:
   0000-01-01: 25_000
+  2023-01-01: 30_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml:4]: this should still be 25,000

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml [hunk 12]
```diff
@@ -1,9 +1,13 @@
 description: Oregon reduces its Child Tax Credit for filers with Oregon adjusted gross income exceeding this amount.
 values:
   0000-01-01: 25_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/start.yaml:3]:
```suggestion
  2023-01-01: 25_000
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/ineligible_age.yaml [hunk 13]
```diff
@@ -3,6 +3,7 @@ values:
   0000-01-01: 6
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/ineligible_age.yaml:3]:
```suggestion
  2023-01-01: 6
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/child_limit.yaml [hunk 14]
```diff
@@ -3,6 +3,7 @@ values:
   0000-01-01: 5
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/child_limit.yaml:3]:
```suggestion
  2023-01-01: 5
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml [hunk 15]
```diff
@@ -4,6 +4,7 @@ values:
   2023-01-01: 1_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml:4]: delete the 0 line

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/width.yaml [hunk 16]
```diff
@@ -3,6 +3,7 @@ values:
   0000-01-01: 5_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/reduction/width.yaml:3]:
```suggestion
  2023-01-01: 5_000
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml (hunk 9 above)
Comment (jialyu0411) [reply policyengine_us/parameters/gov/states/or/tax/income/credits/ctc/amount.yaml:10]: > same question for all ctc files

In 2023 tax forms don't talk about this house bill, but the link was already here before I took over. Should I remove it?

**Comment by PavelMakarchuk** [comment]
Could you add a comment to the issue explaining all of the programs that changed values in 2023?

**Comment by jialyu0411** [comment]
Because there are a lot of changes in the Oregon 2023 parameters, I've summarized them in a Google Doc. Here's the link: [Oregon 2023 parameter values](https://docs.google.com/document/d/12oBX9UHRJzOadRkyq2CgrSezf2p1liFaF4Fp_NE8xSU/edit?usp=sharing)

File: policyengine_us/parameters/gov/states/or/tax/income/rates/head_of_household.yaml [hunk 17]
```diff
@@ -2,15 +2,20 @@ description: Oregon taxes income at these rates for individuals filing as head o
...
   reference:
     - title: Chapter 316 - Personal Income Tax
       href: https://www.oregonlegislature.gov/bills_laws/ors/ors316.html #3160.037 (1)(a)
     - title: 2021 Oregon Income Tax Form OR-40 Instructions
       href: https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2021.pdf#page=27
     - title: 2022 Publication OR-40-NP, Oregon Income Tax Part-year Resident/Nonresident Forms and Instructions, 150-101-048-1
       href: https://www.oregon.gov/dor/programs/individuals/Documents/Part-year%20and%20nonresident,%20Form%20OR-40-P%20and%20OR-40-N%20filers.pdf.pdf
+    - title: 2023 Oregon Income Tax Form OR-40 Instructions
+      href: https://www.oregon.gov/dor/forms/FormsPubs/form-or-40-inst_101-040-1_2023.pdf#page=24
+      # tax rate table as reference: https://www.efile.com/oregon-tax-brackets-rates-and-forms/
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/or/tax/income/rates/head_of_household.yaml:17]: and below

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by PavelMakarchuk** [comment]
[Changes doc](https://docs.google.com/document/d/12oBX9UHRJzOadRkyq2CgrSezf2p1liFaF4Fp_NE8xSU/edit)

File: policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/match/has_young_child.yaml [hunk 18]
```diff
@@ -3,11 +3,14 @@ values:
   2001-01-01: 0.12
 metadata:
   unit: /1
-  label: OR EITC older child age
+  period: year
+  label: Oregon EITC older child age
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/match/has_young_child.yaml:7]:
```suggestion
  label: Oregon EITC match for filers with young children
```

File: policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/old_child_age.yaml [hunk 19]
```diff
@@ -3,11 +3,14 @@ values:
   2001-01-01: 3
 metadata:
   unit: year
-  label: OR EITC maximum age cutoff for higher match
+  period: year
+  label: Oregon EITC maximum age cutoff for higher match
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/or/tax/income/credits/eitc/old_child_age.yaml:7]:
```suggestion
  label: Oregon EITC young child age limit
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**